ophyd/signal.py

```diff
@@ -183,17 +177,8 @@ def describe_configuration(self):
         return self.describe()
 
 
-class EpicsSignal(Signal):
-    '''An EPICS signal, comprised of either one or two EPICS PVs
-
-    =======  =========  =====  ==========================================
-    read_pv  write_pv   rw     Result
-    =======  ========   ====   ==========================================
-    str      None       True   read_pv is used as write_pv
-    str      None       False  Read-only signal
-    str      str        True   Read from read_pv, write to write_pv
-    str      str        False  write_pv ignored.
-    =======  ========   ====   ==========================================
+class EpicsSignalBase(Signal):
```

Comment:
*bikeshed* maybe name this `EpicsSignalRead` and get rid of `EpicsSignalRO` as that way the naming makes sense (`EpicsSignal` is a `EpicsSignalRead` vs `EpicsSignalRW` is a `EpicsSignalReadOnly` (wtf!) ) *\bikeshed*